submissions/ArthurGorbenko/HTML5-Form/css/styles.css

```diff
@@ -0,0 +1,1223 @@
+  background: url("../images/icon-table.png");
+  display: block;
+  background-size: 27px;
+  width: 25px;
+  height: 25px;
+  margin: 0 0 -1.3em -2.5em;
+}
+.form-header_info_table {
+  margin: 1em 0 0;
+}
+.form-header_status_table_cell {
+  margin: 1.5em;
+  padding: 10px 35px 10px 35px;
+  text-align: left;
+}
+.wrapper_form-header_status_content {
+  display: flex;
+  flex-direction: column;
+}
+.wrapper_form-header_status_buttons {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.form-header_info {
+  flex: 1;
+}
+.form-header_info_title {
+  text-align: left;
+}
+.form-header_info_title::before {
+  background-image: url("../images/rescue.svg");
+  background-size: 30px;
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  position: absolute;
+  content: "";
+  margin: -0.5em 0 0 -2.5em;
+}
+.wrapper_form-header_info_content {
+  margin: 1em 6em 1em;
+}
+
+.wrapper_form-header_info_contacts {
+  margin-top: 1.5em;
+  display: flex;
+  justify-content: space-between;
+}
+
+.form-header_submit_button {
+  margin: 1em 0 1em;
+}
+
+.form-header_print_button {
+  margin-bottom: 1em;
+  background: #fff;
+}
+
+.form-header_radio {
+  text-align: center;
+}
+
+.form-header_radio_text {
+  text-transform: uppercase;
+  display: inline-block;
+  margin: 2em;
+  width: 700px;
+  font-size: 2em;
+  text-align: center;
+  border-bottom: 1px solid #ccc;
+  text-align: center;
+}
+
+.form-radio_choice_type {
+  display: flex;
+  justify-content: space-around;
+}
+.form-radio_choice_type_label {
+  outline: 1px solid #cccccc;
+  height: 260px;
+  width: 350px;
+  display: block;
+  text-align: center;
+}
+/* .form-radio_choice_type_button {
+  outline: 1px solid #cccccc;
+  height: 260px;
+  width: 350px;
+  display: block;
+  text-align: center;
+} */
+.form-radio_choice_type_label:hover {
+  outline: 2px solid #6ea9df;
+  cursor: pointer;
+}
+
+.form-radio_choice_type_image {
+  height: 180px;
+  width: 160px;
+  margin: 1em auto;
+}
+.form-radio_choice_type_button {
+  display: flex;
+  flex-direction: column;
+}
+
+input[type="radio"]:checked + .form-radio_choice_type_label {
+  outline: 5px solid #6ea9df;
+}
+input[type="radio"]:checked + .form-radio_choice_type_label::after {
+  content: "";
+  background: url("../images/checked.png") 100% 50% no-repeat;
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin: -16em 0 0 21em;
+  z-index: 9999;
+  position: absolute;
+}
+
+.form-main_header_underline {
+  width: 500px;
+}
+
+.form-upload {
+  display: flex;
+  flex-direction: column;
+  margin: 1em 0;
+}
+
+.form-sections_headers_text {
+  font-size: 28px;
+  margin: 2em 0 2em 40px;
+}
+.form-sections_headers_text::before {
+  content: "1";
+  width: 50px;
+  height: 50px;
+  border: 1px solid #707790;
+  border-radius: 50px;
+  position: absolute;
+  margin: -10px 50px 0 -70px;
+  text-align: center;
+  line-height: 50px;
+}
+
+.form-upload_header::after {
+  content: "(Doc,Docx,Text,PDF,Markdown)";
```

Comment:
This text must be accessible for editing from HTML.

Reply:
Done.